src/stats/show_stats.py

```diff
@@ -49,8 +64,10 @@ async def show_stats(ctx, stat_id):
         make_percentage_graph(
             stat_id, options, answers, stat_title, graph_size=graph_size
         )
-
-        await send_graph_to_channel(ctx, stat_id, stat_title)
+        if imgur_url := await upload_graph_to_imgur_or_none(ctx, stat_id):
```

Comment:
This whole `if` could be moved out of the `PERCENTAGE_BAR` if. But maybe we can worry about that later.